verticaltabs.jsm

```diff
@@ -465,6 +502,37 @@ VerticalTabs.prototype = {
     });
   },
 
+  clearFind: function () {
+    this.document.getElementById('find-input').value = '';
+    this.filtertabs();
+  },
+
+  filtertabs: function () {
+    let document = this.document;
+    let tabs = this.document.getElementById('tabbrowser-tabs');
+    let find_input = this.document.getElementById('find-input');
```

Comment:
Let's change this `this.document` to just use the `document` variable defined above.